R/find_latest_file.R

```diff
@@ -13,9 +13,9 @@
 #' @export
 #'
 #' @examples
-#' find_latest_file(spd_dir, regexp = "Scottish_Postcode_Directory_.+?\\.rds", recurse = TRUE)
-find_latest_file <- function(dir, ...) {
-  fs::dir_info(path = dir, type = "file", ...) %>%
+#' find_latest_file(get_lookups_dir(), regexp = "Scottish_Postcode_Directory_.+?\\.rds")
+find_latest_file <- function(dir, ..., recurse = TRUE) {
+  fs::dir_info(path = dir, type = "file", ..., recurse = recurse) %>%
```

Comment:
Discussed this with Jen.

The example was my mistake and is now fixed!

We also thought having `recurse = TRUE` be the default made sense, so by default it will look in all sub-folders.